### DISCUSSION TITLE
生活的后花园

### DISCUSSION THREAD

**Discussion by shenweiyan** [shenweiyan/Digital-Garden#76] [2.1-乱弹]
年纪越大，越期待短暂的一些独处时光，安静且惬意。

不知道大家的后花园都是什么，在 V2EX 看到，对有些人来说，备菜+做饭的一个小时，不接收外界信息，完全沉浸在只有自己的世界里，是他近几年找到了最好的放松方式了。对我自己可能就在于游戏+看电影，但是能有这样时间的机会确实不多。

看到有人说家里玩游戏会有压力，这一点是同意的，尤其有了娃以后，放空自己的机会就更少了。所以，建立自己的精神后花园，去运运动其实也是一个不错的选择。

**Comment by immelon0097** [comment]
其实我们需要的是随心所欲的生活片段。
